kernel/build-kernel.sh

```diff
@@ -51,6 +51,10 @@ hypervisor_target=""
 arch_target=""
 #
 kernel_config_path=""
+#Force generate config when setup
+force_setup_generate_config="false"
+#Nvidia kernel support
```

Comment:
Is this Nvidia specific, or does it also work for other GPUs?
We should either fix the comment, or change the variable name I suspect.

Reply:
I think  `GPU kernel support` sounds more generic

Reply:
updated. support Intel and Nvidia GPU, `-g nvidia`, `-g intel`